README.md

```diff
@@ -14,8 +20,55 @@ $ ./deploy-local.sh
 This will connect to a dummy database in Heroku. To use a local database, setup the database locally and modify the database configuration/credentials in `deploy-local.sh` accordingly.
 
 **Note.** When deploying locally, the base url is localhost:8080/ and the current subpages are localhost:8080/search, localhost:8080/statistics, localhost:8080/settings and localhost:8080/insert.
```

Comment:
Replace line 22 with: 
**Note.** The application is deployed to an embedded Tomcat container running on port 8080 so the root webpage can be accessed via `localhost:8080`.